### ISSUE TITLE
Stacked minecarts continue momentum when not on a rail

### ISSUE THREAD

**Issue by tryashtar** [tryashtar/minecraft-bugs#14]
## Mojira Ticket: [MC-14](https://bugs.mojang.com/browse/MC-14) by [drgabi18](https://bugs.mojang.com/secure/ViewProfile.jspa?name=drgabi18)

There it's a bug wich let you to make minecarts to walk/fly in direction every direction

To do it, follow the steps:
1.Make the structures as shown in the pictures
2. Place the minecarts like in the picture shown down (To be more easy, put it on the way up rail.).
3. Destroy the block
4. The minecarts will walk

This glitch happen because there are minecarts staked in one.
### Attachments:
  <img src="https://bugs.mojang.com/secure/attachment/10624/2012-10-24_17.21.13.png" width="240" height="135">  <img src="https://bugs.mojang.com/secure/attachment/94319/2015-07-29_15.55.23.png" width="240" height="135">  <img src="https://bugs.mojang.com/secure/attachment/105911/2015-12-01_20.20.12.png" width="240" height="135">  <img src="https://bugs.mojang.com/secure/attachment/10625/Minecarts%20bug.png" width="240" height="135">

**Comment by tryashtar** [comment]
### [Mojira Comment](https://bugs.mojang.com/browse/MC-14?focusedCommentId=15714#comment-15714) by [takatino](https://bugs.mojang.com/secure/ViewProfile.jspa?name=takatino):
Since mine carts can push other mine carts, it would cause an infinite momentum if they are inside each other. One will push the other, and the one pushed will push the other, etc. This can possibly be fixed if they decrease the hit box of their mine carts.

**Comment by tryashtar** [comment]
### [Mojira Comment](https://bugs.mojang.com/browse/MC-14?focusedCommentId=18972#comment-18972) by [scum](https://bugs.mojang.com/secure/ViewProfile.jspa?name=scum):
This bug has been around for a LONG time.

**Comment by tryashtar** [comment]
### [Mojira Comment](https://bugs.mojang.com/browse/MC-14?focusedCommentId=20429#comment-20429) by [tnt](https://bugs.mojang.com/secure/ViewProfile.jspa?name=tnt):
Is this still a concern on Minecraft 1.4.4 Pre-Release?  If so, please update the version affected.

**Comment by tryashtar** [comment]
### [Mojira Comment](https://bugs.mojang.com/browse/MC-14?focusedCommentId=25719#comment-25719) by [mortvert](https://bugs.mojang.com/secure/ViewProfile.jspa?name=mortvert):
Can you replicate this issue in latest version of the minecraft? If so, can you please update the ticket?

**Comment by tryashtar** [comment]
### [Mojira Comment](https://bugs.mojang.com/browse/MC-14?focusedCommentId=34150#comment-34150) by [renex](https://bugs.mojang.com/secure/ViewProfile.jspa?name=renex):
Easily reproducible in 13w01b and any other version :/

**Comment by tryashtar** [comment]
### [Mojira Comment](https://bugs.mojang.com/browse/MC-14?focusedCommentId=34253#comment-34253) by [kaasinees](https://bugs.mojang.com/secure/ViewProfile.jspa?name=kaasinees):
How is decreasing the hitbox going to fix it? The hit boxen is too small on the minecart in the first place. The way to fix is this to apply some error margin in the equation around the borders of the hitbox(also correct the hitbox).

**Comment by tryashtar** [comment]
### [Mojira Comment](https://bugs.mojang.com/browse/MC-14?focusedCommentId=34329#comment-34329) by [renex](https://bugs.mojang.com/secure/ViewProfile.jspa?name=renex):
Maybe adding a little jitter to the entity ejection code?

**Comment by tryashtar** [comment]
### [Mojira Comment](https://bugs.mojang.com/browse/MC-14?focusedCommentId=45784#comment-45784) by [im_orange](https://bugs.mojang.com/secure/ViewProfile.jspa?name=im_orange):
Confirmed in 13w07a

**Comment by tryashtar** [comment]
### [Mojira Comment](https://bugs.mojang.com/browse/MC-14?focusedCommentId=49702#comment-49702) by [pizza2004](https://bugs.mojang.com/secure/ViewProfile.jspa?name=pizza2004):
Can we get a confirmation of whether this happens in 13w10a and an update to the affected versions?

**Comment by tryashtar** [comment]
### [Mojira Comment](https://bugs.mojang.com/browse/MC-14?focusedCommentId=57736#comment-57736) by [powert00l](https://bugs.mojang.com/secure/ViewProfile.jspa?name=powert00l):
The glitch everyone is seeing is the exact same that made boosters work way back when, when the one cart's mesh intersected the other cart's mesh, causing the physics engine to accelerate both carts.

**Comment by tryashtar** [comment]
### [Mojira Comment](https://bugs.mojang.com/browse/MC-14?focusedCommentId=63105#comment-63105) by [tonkku107](https://bugs.mojang.com/secure/ViewProfile.jspa?name=tonkku107):
This is a wierd bug, but is it useful?

**Comment by tryashtar** [comment]
### [Mojira Comment](https://bugs.mojang.com/browse/MC-14?focusedCommentId=65714#comment-65714) by [aerotactics](https://bugs.mojang.com/secure/ViewProfile.jspa?name=aerotactics):
This happens all the way up to 13w18c, I hadn't known this was found before.

That's it, I'm never posting bugs again lol, always being duplicated...

ALSO, easier way to do it: Place ONE track, spawn many carts on it, touch carts.

**Comment by tryashtar** [comment]
### [Mojira Comment](https://bugs.mojang.com/browse/MC-14?focusedCommentId=75761#comment-75761) by [judahMan](https://bugs.mojang.com/secure/ViewProfile.jspa?name=judahMan):
You don't need a ramp, you can just stack minecarts on a one block track, then push them off. If are pushed off a high tower, they will gently float down, slower, and more horizontal then a regular minecart.

**Comment by tryashtar** [comment]
### [Mojira Comment](https://bugs.mojang.com/browse/MC-14?focusedCommentId=75763#comment-75763) by [judahMan](https://bugs.mojang.com/secure/ViewProfile.jspa?name=judahMan):
When I did it, they had little momentum when pushed and actualy gained momentum.

**Comment by tryashtar** [comment]
### [Mojira Comment](https://bugs.mojang.com/browse/MC-14?focusedCommentId=92979#comment-92979) by [StidderDude](https://bugs.mojang.com/secure/ViewProfile.jspa?name=StidderDude):
I think this should stay a future

**Comment by tryashtar** [comment]
### [Mojira Comment](https://bugs.mojang.com/browse/MC-14?focusedCommentId=105572#comment-105572) by [JAMLENOM BACON](https://bugs.mojang.com/secure/ViewProfile.jspa?name=JAMLENOM%20BACON):
I dont want this to go cuz i wall doing this on a really high extreme mountin in the snapshot and it flew of the mountin with me in it and i started to glide foward and slowly decend a i was going in a directions fast and slowly decending and i was bossted up by the minecarts slowly falling off but when they did they pushed me up.
so basically this is one of the only planes without mods in minecraft!

**Comment by tryashtar** [comment]
### [Mojira Comment](https://bugs.mojang.com/browse/MC-14?focusedCommentId=115524#comment-115524) by [GoodKingFilms](https://bugs.mojang.com/secure/ViewProfile.jspa?name=GoodKingFilms):
Still in 1.7.2
Can a Mod update the affected versions? Thanks.

**Comment by tryashtar** [comment]
### [Mojira Comment](https://bugs.mojang.com/browse/MC-14?focusedCommentId=131816#comment-131816) by [jeuvke](https://bugs.mojang.com/secure/ViewProfile.jspa?name=jeuvke):
Confirmed in 14w02c.

**Comment by tryashtar** [comment]
### [Mojira Comment](https://bugs.mojang.com/browse/MC-14?focusedCommentId=145786#comment-145786) by [uncopyrighttnt](https://bugs.mojang.com/secure/ViewProfile.jspa?name=uncopyrighttnt):
confirmed in 14w10c

**Comment by tryashtar** [comment]
### [Mojira Comment](https://bugs.mojang.com/browse/MC-14?focusedCommentId=179631#comment-179631) by [ezekielelin](https://bugs.mojang.com/secure/ViewProfile.jspa?name=ezekielelin):
Is this still a concern in the latest Minecraft version *14w30c*? If so, please update the affected versions in order to best aid Mojang ensuring bugs are still valid in the latest releases/pre-releases.

**Comment by tryashtar** [comment]
### [Mojira Comment](https://bugs.mojang.com/browse/MC-14?focusedCommentId=179721#comment-179721) by [tonkku107](https://bugs.mojang.com/secure/ViewProfile.jspa?name=tonkku107):
Confirmed in 14w30c. It is harder to place them, but it's possible to summon the minecarts on the rail and push them.

**Comment by tryashtar** [comment]
### [Mojira Comment](https://bugs.mojang.com/browse/MC-14?focusedCommentId=200592#comment-200592) by [TheMeaningOfBlah](https://bugs.mojang.com/secure/ViewProfile.jspa?name=TheMeaningOfBlah):
Still in 1.8

**Comment by tryashtar** [comment]
### [Mojira Comment](https://bugs.mojang.com/browse/MC-14?focusedCommentId=204135#comment-204135) by [rydian](https://bugs.mojang.com/secure/ViewProfile.jspa?name=rydian):
Still doable in 1.8.1-pre2 via dropping the minecarts from above, they'll stack and when stacked they'll push themselves until blocked by an obstacle.

**Comment by tryashtar** [comment]
### [Mojira Comment](https://bugs.mojang.com/browse/MC-14?focusedCommentId=209820#comment-209820) by [TheMeaningOfBlah](https://bugs.mojang.com/secure/ViewProfile.jspa?name=TheMeaningOfBlah):
Still in 1.8.1

**Comment by tryashtar** [comment]
### [Mojira Comment](https://bugs.mojang.com/browse/MC-14?focusedCommentId=212301#comment-212301) by [TheMeaningOfBlah](https://bugs.mojang.com/secure/ViewProfile.jspa?name=TheMeaningOfBlah):
Still in 1.8.2-pre1

**Comment by tryashtar** [comment]
### [Mojira Comment](https://bugs.mojang.com/browse/MC-14?focusedCommentId=226159#comment-226159) by [LR G.](https://bugs.mojang.com/secure/ViewProfile.jspa?name=LR%20G.):
Still in 1.8.4

**Comment by tryashtar** [comment]
### [Mojira Comment](https://bugs.mojang.com/browse/MC-14?focusedCommentId=228349#comment-228349) by [emeraldfyr3](https://bugs.mojang.com/secure/ViewProfile.jspa?name=emeraldfyr3):
Still in 1.8.6

**Comment by tryashtar** [comment]
### [Mojira Comment](https://bugs.mojang.com/browse/MC-14?focusedCommentId=228483#comment-228483) by [LR G.](https://bugs.mojang.com/secure/ViewProfile.jspa?name=LR%20G.):
Still in 1.8.5 & 1.8.6

**Comment by tryashtar** [comment]
### [Mojira Comment](https://bugs.mojang.com/browse/MC-14?focusedCommentId=236414#comment-236414) by [husky2490](https://bugs.mojang.com/secure/ViewProfile.jspa?name=husky2490):
Confirmed for Snapshot 15w31a

**Comment by tryashtar** [comment]
### [Mojira Comment](https://bugs.mojang.com/browse/MC-14?focusedCommentId=262337#comment-262337) by [19Spleen](https://bugs.mojang.com/secure/ViewProfile.jspa?name=19Spleen):
Confirmed for 15w45a

**Comment by tryashtar** [comment]
### [Mojira Comment](https://bugs.mojang.com/browse/MC-14?focusedCommentId=264381#comment-264381) by [19Spleen](https://bugs.mojang.com/secure/ViewProfile.jspa?name=19Spleen):
Confirmed for 15w46a

**Comment by tryashtar** [comment]
### [Mojira Comment](https://bugs.mojang.com/browse/MC-14?focusedCommentId=265970#comment-265970) by [19Spleen](https://bugs.mojang.com/secure/ViewProfile.jspa?name=19Spleen):
Confirmed for 15w47a

**Comment by tryashtar** [comment]
### [Mojira Comment](https://bugs.mojang.com/browse/MC-14?focusedCommentId=266429#comment-266429) by [CommanderSealand](https://bugs.mojang.com/secure/ViewProfile.jspa?name=CommanderSealand):
Confirmed for 15w47b

**Comment by tryashtar** [comment]
### [Mojira Comment](https://bugs.mojang.com/browse/MC-14?focusedCommentId=269806#comment-269806) by [CommanderSealand](https://bugs.mojang.com/secure/ViewProfile.jspa?name=CommanderSealand):
Confirmed for 15w49a

**Comment by tryashtar** [comment]
### [Mojira Comment](https://bugs.mojang.com/browse/MC-14?focusedCommentId=273974#comment-273974) by [dominik1411](https://bugs.mojang.com/secure/ViewProfile.jspa?name=dominik1411):
Confirmed for 1.8.8 and 1.8.9
